modules/configuring-your-ide.adoc

```diff
@@ -63,3 +63,60 @@ The extension you installed appears in the *Browser - Installed* list on the *Ex
 
 See link:https://open-vsx.org/[Open VSX Registry] for available third-party extensions that you can consider installing.
 
+ifndef::upstream[]
+== Building the RStudio Server notebook images
+
+To use the *RStudio Server* and *CUDA - RStudio Server* notebook images, you must first build them by creating a secret and triggering the BuildConfig, and then enable them in the {productname-short} UI by editing the `rstudio-rhel9` and `cuda-rstudio-rhel9` imagestreams.
+
+.Prerequisites
+
+* Before starting the RStudio Server build process, you have at least 1 CPU and 2Gi memory available for `rstudio-server-rhel9`, and 1.5 CPUs and 8Gi memory available for `cuda-rstudio-server-rhel9` on your cluster.
+* You are logged in to your OpenShift cluster.
+* You have the `cluster-admin` role in {openshift-platform}, to the namespace `rhoai-applications`, or with cluster-wide role binding.
+* You have an active {org-name} Enterprise Linux (RHEL) subscription.
+
+.Procedure
+
+. Create a secret with subscription manager credentials. These are usually your {org-name} Customer Portal username and password.
```

Comment:
I've seen this referred to as "Red Hat Subscription Manager", "Subscription Manager", "Red Hat Subscription Management". I'd be tempted to capitalize it at least.

Reply:
or take it as commend name and go with `subscription-manager`

Reply:
Have capitalised. Thanks!